src/controllers/activegate/reconciler/statefulset/container_eec.go

```diff
@@ -0,0 +1,110 @@
+package statefulset
+
+import (
+	"fmt"
+
+	"github.com/Dynatrace/dynatrace-operator/src/controllers/activegate/internal/consts"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const eecIngestPortName = "eec-http"
+const eecIngestPort = 14599
+
+const activeGateInternalCommunicationPort = 9999
+
+var _ ContainerBuilder = (*ExtensionController)(nil)
+
+type ExtensionController struct {
+	GenericContainer
```

Comment:
Common initialization & members for ActiveGate-related containers. I remember having bigger refactoring plans around it but since it's not needed much at this point I'll get rid of this extra layer.